OpenStack-accounting/usr/local/sbin/accountinglib.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python3
+import time
+import datetime
+#from datetime import datetime,time
+import json
+import requests
+import sys
+import sqlalchemy
+from sqlalchemy.sql import select
+from sqlalchemy.orm import sessionmaker
+import configparser
+
+def ifnull(var, val):
+    ```Returns the second argument if the first argument is Null/None```
+    if var is None:
+        return val
+    return var
+
+def send_to_influx(datastring):
+    ```Takes a datastring formatted to send to InfluxDBs rest api. Loads necessary config, sends and returns the response```
+     # Read from config file
+     influx_parser = configparser.SafeConfigParser()
+     try:
+         influx_parser.read('/etc/influxdb.conf')
+         host = influx_parser.get('db', 'host')
+         database = influx_parser.get('db', 'database')
+         username = influx_parser.get('auth', 'username')
+         password = influx_parser.get('auth', 'password')
+         instance = influx_parser.get('cloud','instance')
+     except:
+         print('Unable to read from influx config file')
+         sys.exit(1)
+     finaldatastring = datastring.replace("Accounting,","Accounting,instance="+instance+",")
+     print(finaldatastring)
+     url = 'http://'+host+'/write?db='+database +'&precision=s'
```

Comment:
you can use f-strings to make this clearer
```suggestion
     url = f"http://{host}/write?db={database}&precision=s"
```